otherlibs/sites_locations/src/helpers.mli

```diff
@@ -0,0 +1,50 @@
+(** Provide locations information *)
+
+module Location : sig
+  type t = string
+end
+
+module Section : sig
+  type t =
+    | Lib
+    | Lib_root
+    | Libexec
+    | Libexec_root
+    | Bin
+    | Sbin
+    | Toplevel
+    | Share
+    | Share_root
+    | Etc
+    | Doc
+    | Stublibs
+    | Man
+    | Misc
+end
+
+val site :
+     package:string
+  -> section:Section.t
+  -> suffix:string
+  -> encoded:string
+  -> Location.t list
+
+val relocatable : bool Lazy.t
+
+val ocamlpath : string list Lazy.t
+
+val sourceroot : string -> string option
+
+val stdlib : string Lazy.t
+
+val path_sep : string
+
+module HardcodedOcamlPath : sig
```

Comment:
Should be `Hardcoded_ocaml_path`. We use `Pascal_case` everywhere in dune already.